readme.md

```diff
@@ -139,7 +171,15 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+var Instructor = function(name){
+  this.name = name;
+  Instructor.prototype.receivePresent = function(gift){
+    console.log(this.name + " promptly drops the " + gift + " on the floor.")
+  }
```

Comment:
This is all correct, except you should declare prototype methods *outside* (and after) declaring the constructor function they are for:

```js
function Instructor(name){
  this.name = name;
}
Instructor.prototype.receivePresent = function(gift){
  console.log(this.name + " promptly drops the " + gift + " on the floor.");
}

var andy = new Instructor("Andy");
andy.receivePresent("Resin Laying Deer Figurine, Gold");
```